rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java

```diff
@@ -19,6 +21,40 @@
  */
 public final class RxView {
   /**
+   * Create an observable of timestamps for {@code view}'s attaches.
```

Comment:
Yeah all the `Observable<Object>`-returning functions in this class say timestamps :/. What should it say instead?